card/mod/01_core/set/all/fetch.rb

```diff
@@ -58,20 +50,47 @@ def fetch mark, opts={}
         return
       else
         card.include_set_modules unless opts[:skip_modules]  # need to load modules here to call the right virtual? method
-        return unless card.virtual?
+        return unless card.virtual? || opts[:subcard]
       end
-      card.name = mark.to_s if mark && mark.to_s != card.name
+      card.name = mark.to_s if mark && mark.to_s != card.name && !opts[:subcard]
```

Comment:
This feels a bit hacky. Subcards are often new cards so I couldn't fetch them from the cache and I don't want to change the name if I fetch them with a name variant.
Maybe we want a more general solution?
Something in the direction of `Card.fetch 'name', include_new: true` instead of `Card.fetch 'name', subcard: true`?